users/views.py

```diff
@@ -0,0 +1,10 @@
+from . import models, serializers
+from balarco import utils
+
+
+class UserViewSet(utils.GenericViewSet):
+    """ViewSet for User CRUD REST Service that inherits from utils.GenericViewSet
+    """
+    obj_class = models.User
```

Comment:
I treated this ws as any other, maybe I should've done it differently, but I thought RBAC was going to be handled in a different task